VocaDbModel/Service/OtherService.cs

```diff
@@ -275,48 +264,16 @@ private IEnumerable<EntryWithCommentsContract> CreateEntryWithCommentsContract<T
 			return comments.GroupBy(e => e.Entry, new EntryComparer()).Select(e => new EntryWithCommentsContract(entryContractFac(e.First()), e.Select(c => new CommentContract(c)).ToArray()));
 		}
 
-		private async Task<List<Comment>> GetComments<TEntry, TComment>(ISession session, int maxComments, bool checkDeleted) where TComment : GenericComment<TEntry> where TEntry : class, IEntryWithNames
-		{
-			var q = session.Query<TComment>();
-
-			if (checkDeleted)
-				q = q.Where(c => !c.EntryForComment.Deleted);
-
-			return await q.OrderByDescending(c => c.Created).Take(maxComments).Cast<Comment>().VdbToListAsync();
-		}
-
 		private async Task<EntryWithCommentsContract[]> GetRecentCommentsAsync(ISession session, int maxComments)
 		{
-			var albumComments = await GetComments<Album, AlbumComment>(session, maxComments, true);
-			var artistComments = await GetComments<Artist, ArtistComment>(session, maxComments, true);
-			var songComments = await GetComments<Song, SongComment>(session, maxComments, true);
-			var discussionComments = await GetComments<DiscussionTopic, DiscussionComment>(session, maxComments, true);
-			var songListComments = await GetComments<SongList, SongListComment>(session, maxComments, false);

```

Comment:
Why was `checkDeleted` `false` here?

Reply:
Soft-deletion was added to song lists afterwards